src/IB/ConstraintIBMethod.cpp

```diff
@@ -169,6 +169,9 @@ ConstraintIBMethod::ConstraintIBMethod(const std::string& object_name,
       d_FuRMoRP_current_time(0.0),
       d_FuRMoRP_new_time(0.0),
       d_vol_element(d_no_structures, 0.0),
+      d_structure_vol(d_no_structures, 0.0),
+      d_structure_mom(d_no_structures, std::vector<double>(3, 0.0)),
```

Comment:
I was following the convention in this class, where all vector quantities were std::vectors.

Would you prefer I change all vectors in ConstraintIBMethod to IBTK::Vector3d, or just the ones that are needed for the hydro force calculation?